tailwind.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
           800: "#262F5F",
           900: "#192040",
         },
+        textGray: '#68737D'
```

Comment:
Let's use the colors from neetoUI.
Ref - https://neetoui-v2.netlify.app/?path=/story/foundation-helpers-color--page